debian/tests/usage

```diff
@@ -2,5 +2,5 @@
 
 set -ex
 
-ubuntu-advantage --help | grep --silent services
-ubuntu-advantage status | grep --silent 'This machine is not attached'
+ua --help | grep --silent services
+ua version
```

Comment:
Nit: Should we test both spellings of the command?

Reply:
All our docs going forward are going to use "ua". I treat "ubuntu-advantage" as a nice to have for previous users, even though they will have to change their processes anyway. So I'm not heavily inclined to ensure those work.

Thoughts?

Reply:
I don't feel at all strongly about it, but it won't be particularly nice to have for them if it stops working. ;-)